x-pack/plugins/reporting/server/export_types/printable_pdf_v2/types.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SerializableState } from 'src/plugins/kibana_utils/common';
+import { LocatorParams } from '../../../common/types';
+import { LayoutParams } from '../../lib/layouts';
+import { BaseParams, BasePayload } from '../../types';
+
+interface BaseParamsPDFV2<P extends SerializableState = SerializableState> {
+  layout: LayoutParams;
```

Comment:
should we remove layout parameter on v2 ? cc @tsullivan

Reply:
The layout params tell us the width and height, which is fine.

We will still use the layout ID to know if the user has selected "optimize for print." This PR isn't getting rid of the need to support that, but it is another really high priority for us. Maybe that can be phased out within V2 in the future.